torch_frame/data/multi_nested_tensor.py

```diff
@@ -358,6 +358,23 @@ def stack(xs: List['MultiNestedTensor'],
         else:
             raise NotImplementedError
 
+    @staticmethod
+    def cat(xs: Union[List['MultiNestedTensor'], Tuple['MultiNestedTensor']],
+            dim: int = 0) -> 'MultiNestedTensor':
+        # TODO: To be implemented
+        if len(xs) == 1:
+            return xs[0]
+
+        if dim < 0:
+            dim += xs[0].ndim
+
+        if dim == 0:
+            raise NotImplementedError
+        elif dim == 1:
+            raise NotImplementedError
+        else:
```

Comment:
Do we need to support cat on the third dimension? Maybe raise a different error here.

Reply:
We should raise an error.